extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -130,4 +147,13 @@ HealthBuildItem addHealthCheck(KafkaBuildTimeConfig buildTimeConfig) {
         return new HealthBuildItem("io.quarkus.kafka.client.health.KafkaHealthCheck",
                 buildTimeConfig.healthEnabled, "kafka");
     }
+
+    @BuildStep
+    AdditionalBeanBuildItem addRuntimeConfig() {
+        final Builder builder = AdditionalBeanBuildItem.builder()
```

Comment:
I believe we should be able to avoid this new bean. 
It seems to be convenient for the tests, but the Kafka Test Resource can configure properties (that's the Map returned by the `start` method)

Reply:
If we go with this, we would need to adapt the Reactive Messaging Kafka connector to use the same config. This would be a breaking change.